code/game/machinery/portable_turret.dm

```diff
@@ -1022,7 +1019,10 @@ GLOBAL_LIST_EMPTY(turret_icons)
 	syndicate = TRUE
 	installation = null
 	always_up = TRUE
-	use_power = NO_POWER_USE
+	requires_power = FALSE
+	power_state = NO_POWER_USE
+	idle_power_consumption = 0
+	active_power_consumption = 0
```

Comment:
```suggestion
```
Machinery already sets both as 0, the power_state being no_power_use should be enough to indicate that it doesnt use power. Same happens at beacon.dm and atmospheric.dm